committee-steering/governance/sig-governance.md

```diff
@@ -82,9 +82,22 @@ Subproject contributors (as applicable).
 
 ### Chair
 
+From governance.md:
+> SIG Chairs are intended to be organizers and facilitators, responsible for 
+> the operation of the SIG and for communication and coordination with the 
+> other SIGs, the Steering Committee, and the broader community. 
+
 - Number: 2-3
+- Term: 3 years
+  - 90 days before term is up, Chair will send an email to the SIG mailing list,
+    announce at the regularly scheduled meeting, and send a note to the slack
+    channel asking for interested individuals to step forward. This provides
+    time for outreach, training, and onboarding. 
```

Comment:
what is the process if more than one candidate steps up?

Reply:
In sig instrumentation we have used cordecet voting.

Reply:
Voting systems have to define who has standing to vote, how did you decide that?

Reply:
We try to keep [this up-to-date](https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-instrumentation/teams.yaml#L21-L43).

Reply:
By membership, I assume you mean active contributors/reviewers @logicalhan? For SIGs like ContribEx, Docs, and Release, this approach might not feasibly scale, IMO. If we choose to go down the voting route, I assume we'd need to refer to Devstats data just like we do with Steering elections.

Reply:
I mean, we use devstats to populate our lists, so effectively, they are kinda the same thing.